### PR TITLE
Chore merge from d8883d7 to 8e5dada

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/d8883d753f45d819234b4533b58da161c4ab1e8b
- [x] https://github.com/twbs/bootstrap/commit/2f617215755b066904248525a8c56ea425dde871 
- [x] https://github.com/twbs/bootstrap/commit/5036f70e1c80cebb270aca8c1b422f8d4d4bd47f
- [x] https://github.com/twbs/bootstrap/commit/e4741f7e1e234ef41c119020fb16ea0661366012
- [x] https://github.com/twbs/bootstrap/commit/f722615d35b6d2b0348ca19af054c2ec39755e51
- [x] https://github.com/twbs/bootstrap/commit/3a8c7587a33926e3253730c103650fd098f415fa
- [x] https://github.com/twbs/bootstrap/commit/e946bf7e9434a99d750baf9a0786fcf52c5ff8f7
- [x] https://github.com/twbs/bootstrap/commit/8321397656762cf2007bcf028de1b938ce58b058
- [x] https://github.com/twbs/bootstrap/commit/9369cd93fb177a967ae3d59485721491bb388d1d
- [x] https://github.com/twbs/bootstrap/commit/14292b4a22b3390a633184b97043e3e4a85db93a
- [x] https://github.com/twbs/bootstrap/commit/213888e17e6f9fe9ea7ae67e9d9c1c3e8494867a
- [x] ~~https://github.com/twbs/bootstrap/commit/6dc18c6adccc1edef22aeaf22e21bce29dc740f0~~: nothing to do here because this example is hidden in Boosted doc (tracked in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/2140 to be reintegrated with the dark mode support)
- [x] https://github.com/twbs/bootstrap/commit/787dfea8a29db138d5ff6100fc894f457ccd4169
- [x] https://github.com/twbs/bootstrap/commit/221862fe9447bce74e383e3ee1cbd8a116c4f499
- [x] https://github.com/twbs/bootstrap/commit/e244dbc9d8efa8827d24d4f95651e73804c3216d
- [x] https://github.com/twbs/bootstrap/commit/5370ddc4d336fc7f1810e8b0e35e1562da49e897
- [x] https://github.com/twbs/bootstrap/commit/88c488515f6e1a81b5ac3ce5941a78b4e73384b6
- [x] https://github.com/twbs/bootstrap/commit/d333d70e52dd8b13d043a0c11ccef9282a27ccd4
- [x] https://github.com/twbs/bootstrap/commit/1ab75c8d14b101d7f2ab8d31f0addf549c70def7
- [x] https://github.com/twbs/bootstrap/commit/43e4bf97a8b892af42f5a03ce7056efb6efa3b58
- [x] https://github.com/twbs/bootstrap/commit/e0d54ffd3202b7f6a6b2b738ca6ec15017748925
- [x] https://github.com/twbs/bootstrap/commit/4315801856b690d30091e804fb1de5d6233fe306: applied it as-is but I'm going to create a PR upstream to avoid loading Algolia files when unit examples are rendered
- [x] https://github.com/twbs/bootstrap/commit/e2854b94da6ead85753c9b9816b2ffdf3964a882
- [x] https://github.com/twbs/bootstrap/commit/52d53696d25782ecfa10930d5a297b60e2fc445e
- [x] https://github.com/twbs/bootstrap/commit/3515ac16a171ca4bbb131237ff3324a6b0eb8743
  - ⚠️ This one wasn't easy to merge, please double-check thoroughly
- [x] https://github.com/twbs/bootstrap/commit/6301ca1ee070b1614ff1fd9fc23b0f1bf869475e
- [x] ~~https://github.com/twbs/bootstrap/commit/2c7f888c1ec667119c6a811ed247aaa3cd539fed~~:
  - Too complex to reintegrate in this PR. Created https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/2163 to tackle it separately.
- [x] https://github.com/twbs/bootstrap/commit/bd7a0452c07aaf665f778afa50988b1fbcc8088a
- [x] https://github.com/twbs/bootstrap/commit/fef8958209accdc9e2117ba6d67446aadc2c6425
- [x] https://github.com/twbs/bootstrap/commit/609e6457cc5706fb5de439edc5073641b1a695d5
- [x] https://github.com/twbs/bootstrap/commit/df382155853952de8a2663774c63c8088f2c89b6
- [x] https://github.com/twbs/bootstrap/commit/42a055c7a8e611523c2d10650a465d75d0bb362b
- [x] https://github.com/twbs/bootstrap/commit/4617a7610cde5b3912cb01c8e3471368d9dd1f5e
- [x] https://github.com/twbs/bootstrap/commit/5bcb0fafad4e69ac437f6b4c358b485d13de42b3
- [x] https://github.com/twbs/bootstrap/commit/79e196a9760456e5d3b4e30d9212cda280de5d72
- [x] https://github.com/twbs/bootstrap/commit/9ff96a497ca005e4028a438e2f6838756a8a0cb1
- [x] https://github.com/twbs/bootstrap/commit/e2de56d74038077d36bd1edf7f45f7d299f0414a
- [x] https://github.com/twbs/bootstrap/commit/2990534463f2ed7c9ba378d091658fb1b1f18808
- [x] https://github.com/twbs/bootstrap/commit/afa2fc447f0ab22402715e1ab92a13bf67ec1732
- [x] https://github.com/twbs/bootstrap/commit/de69649b3aca7bbe2eb6b6fb294cb508d8d6ad04
- [x] https://github.com/twbs/bootstrap/commit/a5c7303fa23a2c88d930c233b26da4643aeba5d9
- [x] https://github.com/twbs/bootstrap/commit/a0931d0f6b43d36a95840237979ba6ee2b447a01
- [x] https://github.com/twbs/bootstrap/commit/ccc73058c1e1bcd5fdab971b4073eaf5dbe1523a
- [x] https://github.com/twbs/bootstrap/commit/877e6725e679007c93c1e5663d209216807fb3ee
- [x] ~~https://github.com/twbs/bootstrap/commit/cdcb503b88a6d3ddcc870022a740b996af0034d3~~: will regenerate it at the end of this PR if possible
- [x] https://github.com/twbs/bootstrap/commit/8c135169e7ef5a4d6b92b99e21630711d7830963
- [x] https://github.com/twbs/bootstrap/commit/6a9b9af59e0e279e3fc6eb46a043f9a4eae578fe
- [x] https://github.com/twbs/bootstrap/commit/674ea99c8c70d8833bb2437d1cd3d098af8c5edb
- [x] https://github.com/twbs/bootstrap/commit/7dfa9c58902eaa05ba65945dd6f3ca6c501b7a3e
- [x] https://github.com/twbs/bootstrap/commit/3ea4168cb62f40b1ee3801d827c59333e5996bfb
- [x] ~~https://github.com/twbs/bootstrap/commit/b3da789113c8d17bfe4650d8fe6f6cf80436a1cd~~: we don't have any Cheatsheet example pages in Boosted
- [x] ~~https://github.com/twbs/bootstrap/commit/101ff73b4ca2c657644342eafc2677c27f711038~~: we don't have any Carousel example pages in Boosted
- [x] https://github.com/twbs/bootstrap/commit/9205ffe6311dc20578f6b6b77757f8955867d199
- [x] https://github.com/twbs/bootstrap/commit/303e56758d9f531699aa6703eea8c032ad2525d2
- [x] https://github.com/twbs/bootstrap/commit/cf074cd3b95d9cc08fc8cc8725e80327d5f3071f
- [x] https://github.com/twbs/bootstrap/commit/ae255aaefc6a1fcd458429b03bf579e19e347943
- [x] https://github.com/twbs/bootstrap/commit/83e5ec81752148e6b4e0a9d31053f840fbad23ec
- [x] https://github.com/twbs/bootstrap/commit/fb348b6bddb99cbb73554ecf42e63b0ecfaab90b
- [x] https://github.com/twbs/bootstrap/commit/86c5371ac8da1636f0e8406f62e3a1610cad4b0f
- [x] https://github.com/twbs/bootstrap/commit/fd0a4e67a01162d39bf1844ec70ae3e8cc022102
- [x] https://github.com/twbs/bootstrap/commit/a814733ad749ff73f4f3534b37ccee97d3887029
- [x] https://github.com/twbs/bootstrap/commit/8fcfce1ebd988fd4e5ed2f51acaa5c49e6b301c9
- [x] https://github.com/twbs/bootstrap/commit/d45cc7ef516de676bfaa2b97fe10b71e98ff6f3e
- [x] https://github.com/twbs/bootstrap/commit/1c80bd9c4b3929c51f6d654ab1f1d721b6212e47
- [x] https://github.com/twbs/bootstrap/commit/4271d22b902e8aa2c56c22c39737930946ecf6b9
- [x] ~~https://github.com/twbs/bootstrap/commit/01dacaf2189b5b4319bd172cba7f10d22cb860e6~~:  we don't have any Dashboard example pages in Boosted
- [x] https://github.com/twbs/bootstrap/commit/92d779f83c265a447b72d0e6df2f0e68c3d1fb40
  - ⚠️ Also regenerated `package-lock.json` at this moment 
- [x] https://github.com/twbs/bootstrap/commit/ea21de16740d8eb4f1bdccbb7db8860d8ba36c7a
- [x] ~~https://github.com/twbs/bootstrap/commit/cc5a8a903612e9c0de92238ac17dcb083efce138~~: we don't have floating labels in Boosted
- [x] https://github.com/twbs/bootstrap/commit/57ddb42bafe3cbc4b983fc8acd10b8561218cea0
- [x] ~~https://github.com/twbs/bootstrap/commit/11b3806675de929a2bb342a19b9c65fc92c473a3~~: nothing to do here since our own bundlewatch values were OK
- [x] ~~https://github.com/twbs/bootstrap/commit/8e5dada5b15a7605c856a05005cc3206cd4a72e3~~: we don't have any Buttons example pages in Boosted

### CODE REVIEW

**Comment by julien-deramond** [comment]
@louismaximepiton I let you review this PR and in the meantime I'll check why pa11y's not happy (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/runs/5656026746/job/15322504889?pr=2162).

File: site/content/docs/5.3/components/button-group.md [hunk 1]
```diff
@@ -228,9 +226,7 @@ These 2 vertical variants should not be used because they do not respect the Ora
       <li><a class="dropdown-item" href="#">Dropdown link</a></li>
     </ul>
   </div>
-  <button type="button" class="btn btn-secondary">Button</button>
-  <button type="button" class="btn btn-secondary">Button</button>
-  <div class="btn-group" role="group">
+  <div class="btn-group dropstart" role="group">
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/button-group.md:229]: **[Bootstrap Side]** The markup isn't the one shown in the doc for the dropstart.

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
We should maybe add something in the migration guide about the `aria-disabled` thing ?

Should we change the `appearance` attribute for QS too ? https://github.com/twbs/bootstrap/commit/1ab75c8d14b101d7f2ab8d31f0addf549c70def7 I don't know how to test it properly. But changing its place doesn't seem to bring new errors so I'm a bit lost on this.

Few things to tackle before merging.

File: scss/_variables.scss
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/_variables.scss:1]: Good catch! Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162/commits/12fe4835ffb151125ec3417f1f3937063523f69a

File: site/content/docs/5.3/components/navs-tabs.md
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/components/navs-tabs.md:1]: Can't see the context of this message, where?

File: site/content/docs/5.3/components/list-group.md
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/components/list-group.md:1]: Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162/commits/224b13140cdb87d60a25c4901165fdefb72aece5

File: scss/_nav.scss
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/_nav.scss:1]: Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162/commits/be7817ff39547b615aa3ab601eb5ae3d7a919f34 (+ adapted a little our bundlewatch values)

File: site/data/docs-versions.yml
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/data/docs-versions.yml:1]: Damn! Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162/commits/05ebff56ea66fac9018a37f8d55d2ac3298f7081

**Comment by julien-deramond** [comment]
> Should we change the appearance attribute for QS too ? https://github.com/twbs/bootstrap/commit/1ab75c8d14b101d7f2ab8d31f0addf549c70def7 I don't know how to test it properly. But changing its place doesn't seem to bring new errors so I'm a bit lost on this.

Upstream this change has been made to prevent an issue. Between the commits and the deps updates, there was a moment where the issues were actually there; and I had to do this commit https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162/commits/53652a44d91cd002b159e5efc8c70c962a034732. There was no error with the quantity selector so I think we're good. It's reproducible anymore since the deps have changed again in the recent commits.

File: site/content/docs/5.3/components/navs-tabs.md
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/components/navs-tabs.md:1]: Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2162/commits/26ff4caaef9d99306d730c1dab6bb3b5e4122c4d

File: site/content/docs/5.3/components/navs-tabs.md [hunk 2]
```diff
@@ -233,7 +233,7 @@ Nav tabs light is nested in a tab for adding a level of depth in information org
         <a class="nav-link" id="nav-linkC" href="#linkC" data-bs-toggle="tab" data-bs-target="#linkC" role="tab">Link C</a>
       </li>
       <li class="nav-item" role="presentation">
-        <a class="nav-link disabled" id="nav-linkD" data-bs-toggle="tab" data-bs-target="#linkD" role="tab">Link D</a>
+        <a class="nav-link disabled" id="nav-linkD" data-bs-toggle="tab" data-bs-target="#linkD" role="tab"  aria-disabled="true">Link D</a>
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/navs-tabs.md:236]:
```suggestion
        <a class="nav-link disabled" id="nav-linkD" data-bs-toggle="tab" data-bs-target="#linkD" role="tab" aria-disabled="true">Link D</a>
```

**Review by louismaximepiton — APPROVED** [review]
LGTM!

**Pull request merged**